cpp/tests/json/json_tests.cpp

```diff
@@ -588,6 +588,15 @@ TEST_F(JsonPathTests, GetJsonObjectIllegalQuery)
     };
     EXPECT_THROW(query(), std::invalid_argument);
   }
+
+  {
+    cudf::test::strings_column_wrapper input{R"({"a": "b"})"};
+    std::string json_path("${a}");
+    auto query = [&]() {
+      auto result = cudf::get_json_object(cudf::strings_column_view(input), json_path);
+    };
```

Comment:
1. It's better to use `const` everywhere possible.
2. Use `auto const` consistently.
3. In addition, use initializer list `{}` instead of parentheses if possible.
```suggestion
    auto const input = cudf::test::strings_column_wrapper{R"({"a": "b"})"};
    auto const  json_path = std::string{"${a}"};
    auto const query = [&]() {
      auto result = cudf::get_json_object(cudf::strings_column_view{input}, json_path);
    };
```

Reply:
Thanks for the suggestions Nghia, will keep in mind!